components/size_aware_image.jsx

```diff
@@ -114,8 +101,11 @@ export default class SizeAwareImage extends React.PureComponent {
             ...props
         } = this.props;
 
+        let placeHolder;
+        const shouldShowImg = !dimensions || this.state.loaded;
```

Comment:
This is destructed from the props. You mean here in the component? or pass the props as `placeholderDimensions `?

Reply:
For now i merged this. I will address this with another PR. let me know.

Reply:
Thats Kinda of a big change actually because we use this component in lot of places in the app and the dimensions is the key from the post object we destructure. I am inclined to keeping the way it is for now. @hmhealey Thoughts?